src/constants/channels.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const NotificationLevel = {
```

Comment:
Should we start moving towards `enum` like we discussed?

Reply:
I'm leaving this one as-is for now since it's existing field in a poorly-typed data structure (NotifyProps), but I agree with enums for new types going forward.